render/detailed/summary.go

```diff
@@ -103,6 +104,7 @@ var primaryAPITopology = map[string]string{
 	report.PersistentVolume:      "pods",
 	report.PersistentVolumeClaim: "pods",
 	report.StorageClass:          "pods",
+	report.Disk:                  "pods",
```

Comment:
We're planning to show disks in the `host`. So, I think it should be `host` not `pods`